suse/build/SOURCES/jenkins.init.in

```diff
@@ -101,13 +111,21 @@ case "$1" in
         if [ $CHECK -eq 7 ]; then
                 rm -f "$JENKINS_PID_FILE"
                 if [ -x "$JENKINS_INIT_SHELL" ]; then
-                       startproc -n $JENKINS_NICE -s -e -l /var/log/@@ARTIFACTNAME@@.rc -p "$JENKINS_PID_FILE" -t 1 /bin/su -l -s "$JENKINS_INIT_SHELL" -c "$JAVA_CMD $PARAMS &" "$JENKINS_USER"
+                       startproc -n $JENKINS_NICE -s -e -l /var/log/@@ARTIFACTNAME@@.rc -p "$JENKINS_PID_FILE" /bin/su -l -s "$JENKINS_INIT_SHELL" -c "$JAVA_CMD $PARAMS &" "$JENKINS_USER"
```

Comment:
The combination of `-t 1` and `&` starts a process in the background, then waits for 1 second hoping for it to start. A few lines down, we run `pgrep java` expecting the process to be running. If the process happens to launch within 1 second, the service startup is successful. On our test environments, which is a 4-CPU system starting up 15 Jenkins instances in parallel, 1 second is not enough time and the service sometimes fails. Rather than increase the wait time arbitrarily, I've removed waiting entirely and added a loop that sleeps 1 second waiting for the process to launch, waiting up to 30 seconds.